glitter/reminders/models.py

```diff
@@ -0,0 +1,60 @@
+import datetime
+from calendar import monthrange
+
+from dateutil.relativedelta import relativedelta
+
+from django.conf import settings
+from django.contrib.contenttypes.fields import GenericForeignKey
+from django.contrib.contenttypes.models import ContentType
+from django.db import models
+from django.utils import timezone
+
+from . import choices as reminders_choices
+
+
+class Reminder(models.Model):
+    object_id = models.PositiveIntegerField()
+    content_type = models.ForeignKey(ContentType)
+    content_object = GenericForeignKey('content_type', 'object_id')
+    user = models.ForeignKey(settings.AUTH_USER_MODEL)
```

Comment:
Can we add a unique together for user + object_id + content_type?

Prevent any confusion incase users go adding multiple reminders for themselves.